src/renderer/components/kube-object-details/kube-object-detail-items/kube-object-matches-to-kind-and-api-version.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import type { KubeObject } from "../../../../common/k8s-api/kube-object";
+
+export const kubeObjectMatchesToKindAndApiVersion =
+  (kind: string, apiVersions: string[]) =>
+  <TKubeObject extends KubeObject>(item?: TKubeObject) =>
```

Comment:
nit: I think you meant to have `item: TKubeObject | undefined`

Reply:
Thanks.